src/main/java/sirius/biz/tenants/mongo/MongoTenant.java

```diff
@@ -103,6 +104,7 @@ public MongoRef<MongoTenant> getParent() {
     }
 
     @Override
+    @NoodleSandbox(NoodleSandbox.Accessibility.GRANTED)
```

Comment:
put on super-method?

Reply:
The annotations currently don't work on interfaces